Mage/src/main/java/mage/abilities/decorator/ConditionalRequirementEffect.java

```diff
@@ -125,6 +125,16 @@ public boolean mustBlockAny(Game game) {
         return false;
     }
 
+    @Override
+    public boolean mustBlockAllAttackers(Game game) {
```

Comment:
method is `mustBlockAllAttackers` but override is calling `mustBlockAny`

Reply:
Whoops, sorry about that.  Should be fixed now.